comms.py

```diff
@@ -0,0 +1,62 @@
+from mcu import *
+from controls import *
+
+class Communications:
+    def __init__(self, mcuVAR: MCUInterface, controller: controls):
+        self.mcuVAR = mcuVAR
+        self.controller = controls
+
+        MOTOR_CLAW = 0 
```

Comment:
I think these variables have local scope, and so you won't be able to use them later on in the program. I would suggest adding `self.__`: `self` to make the instance own it, and `__` to indicate it as a private variable.

Reply:
It's probably better to move these to the top and use them as constants, or just import `command_constants.py` and use that. Besides, it's somewhat unnecessary to use constants here, since the mapping can be changed on the microcontroller level.

Reply:
Feel free to add some constants to mcu_lib/command_constants.py
```py
PWM_MIN = 1000
PWM_MID = 1500
PWM_MAX = 2000

MOTOR_FRONT = 0
MOTOR_BACK = 1
MOTOR_LEFT = 2
MOTOR_RIGHT = 3
MOTOR_CLAW = 4
```